smartcontract/cli/src/checkversion.rs

```diff
@@ -11,11 +11,11 @@ pub fn check_version<C: CliCommand, W: Write>(
     if let Ok((_, pconfig)) = client.get_program_config(GetProgramConfigCommand) {
         // Compare the program version with the client version
         // If the program version is incompatible, return an error
-        if pconfig.version.error(&client_version) {
+        if client_version < pconfig.min_compatible_version {
             eyre::bail!("Your client version is no longer up to date. Please update it before continuing to use the client.")
```

Comment:
Can we show the client version and the latest program version they should upgrade to in this message too?